### PR TITLE
FEG-231 - feature - tabs

### PR DESCRIPTION
**Pull request by arthurrigley**
## Summary of Changes
1. Add disabled tab
2. Updated and improved overall styling
- Add mobile horizontal scroll (on non admin panel)
- Dark mode styling
5. Update admin panel (small amount to change)
6. Update version control table
7. Add pdf download
8. Update README areas

## Review
Open up the preview site (see the below netlify comment below) and navigate to:
- /components/tabs

## Ticket(s)
FEG-231

## Checklist

The below needs to be done before a pull request can be approved:

- [x] The pull request command has been ran `npm run pull-request`
- [ ] New components work as a Vue component, Web component -and as static HTML-
- [ ] New components added as an export to src/index.js
- [ ] New components/features have sufficient unit tests
- [ ] New components/features have sufficient documentation
- [ ] New component has dataLayer events added
- [ ] New component is added to the rollup config

## Accesibility check list

- [ ] New components/features are accessible to keyboard users (All links/buttons are tabbable, All content is accessible)
- [ ] New components/features are accessible to non-JS users (All links/buttons are tabbable, All content is accessible), this may have visual differences
- [ ] New components/features have hover, focus and active states on all the links/buttons
- [ ] New components/features work when in dark mode and high contrast mode. Buttons, links and icons should still look like what they are.

### CODE REVIEW

File: assets/sass/components/tabs.scss [hunk 1]
```diff
@@ -30,37 +73,45 @@
     font-family: var(--font-heading);
     font-size: rem(18);
     cursor: pointer;
-    margin: 0.5rem 1.5rem 0 0;
-    padding: 0;
+    margin: 0;// 0.5rem 1.5rem 0 0;
```
Comment (JamesLambert1986) [inline assets/sass/components/tabs.scss:76]: Can the comment be removed.

File: docs/views/components/TabsDoc.vue [hunk 2]
```diff
@@ -1,27 +1,85 @@
+<script setup>
+  import DSHeader from '../DSHeader.vue'
+  import headerImg from '../../img/cards-header.png'
+  //import anatomy from ''
```
Comment (JamesLambert1986) [inline docs/views/components/TabsDoc.vue:4]: Remove this line

File: src/components/Tabs/Tabs.vue [hunk 3]
```diff
@@ -9,12 +9,11 @@
 </style>
 
 <script>
-import iamTabs from '../../../assets/js/components/tabs/tabs.component.min.js'
+import iamTabs from '../../../assets/js/components/tabs/tabs.component.js'
```
Comment (JamesLambert1986) [inline src/components/Tabs/Tabs.vue:12]: Can we put the .min back please. I know this is annoying and ill try to come up with dev/production flag or something in the future.

**Review by JamesLambert1986 — CHANGES_REQUESTED** [review]
Can you deal with the very minor things i've added aswell as get an approval from Mark.

It would be good to get the missing content in first aswell then you can take the (beta) text from the routing file.

File: assets/ts/components/tabs/tabs.component.ts [hunk 4]
```diff
@@ -40,6 +42,11 @@ class iamTabs extends HTMLElement {
     
     let classList = this.classList.toString().replace('container','');
     this.shadowRoot.querySelector('.tabs').setAttribute('class',`tabs ${classList}`);
+    
+    // let buttons = this.shadowRoot.querySelector('.link')
```
Comment (natwalker-iap) [inline assets/ts/components/tabs/tabs.component.ts:46]: can these comments be removed? (46-49)

**Review by natwalker-iap — CHANGES_REQUESTED** [review]

**Comment by arthurrigley** [comment]
@JamesLambert1986  @natwalker-iap  - thanks for comments. I've done the following under the latest commit:

- Add the anatomy image
- Remove the beta from route name
- Remove unneccesary comments
- Re-add .min file
- Add new PDF
- Update text from new PDF
- run `npm run pull-request`

**Review by JamesLambert1986 — APPROVED** [review]

**Pull request merged**